.github/workflows/cypress-tests-runner.yml

```diff
@@ -0,0 +1,174 @@
+name: Run Cypress tests
+
+on: 
+  workflow_dispatch:
+  pull_request:
+  merge_group:
+    types: [checks_requested]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CARGO_INCREMENTAL: 1
+  CARGO_NET_RETRY: 10
+  CONNECTORS: stripe
+  RUST_BACKTRACE: short
+  RUSTUP_MAX_RETRIES: 10
+  SKIP_STEP: ${{ ((github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)) || (github.event_name == 'merge_group')}}
```

Comment:
Either the condition needs to be negated or the variable needs to be renamed: this condition includes the condition to run tests, not skip them.